packages/block-editor/src/components/media-placeholder/index.js

```diff
@@ -87,7 +87,7 @@ export function MediaPlaceholder( {
 	}, [ value ] );
 
 	const onlyAllowsImages = () =>
-		allowedTypes.every(
+		( allowedTypes ?? [] ).every(
```

Comment:
Judging from the name, this function should return false if everything is allowed and not true, right?

Reply:
Yes, it should. Thanks for the catch.